src/layouts/default.hbs

```diff
@@ -19,9 +19,9 @@
     <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'en'}}{{url}}" hreflang="x-default" />
 
     <link rel="stylesheet" href="https://infragistics.com/assets/modern/css/layout.css">
-    <link rel="stylesheet" href="https://infragistics.com/assets/modern/css/animate-custom.css">
+<link rel="stylesheet" href="https://infragistics.com/assets/modern/css/animate-custom.css">
     <link rel="stylesheet" href="https://infragistics.com/assets/modern/css/fontello.css">
-    <link rel="stylesheet" href="https://infragistics.com/css/navigation.css">
+    <link rel="stylesheet" href="{{getConfigData 'url' 'en'}}/css/navigation.css">
     <link rel="stylesheet" href="https://infragistics.com/css/footer.css">
     <link rel="stylesheet" href="https://infragistics.com/css/navigation.css">
     <link rel="stylesheet" href="https://infragistics.com/css/footer.css">
```

Comment:
@Aleksandyr I just saw that two of the marked resources are redundant:
<link rel="stylesheet" href="https://infragistics.com/css/navigation.css">
and 
<link rel="stylesheet" href="https://infragistics.com/css/footer.css">

What are your thoughts?

Reply:
Can't determine which resources your pointing to? You are pointing the one that you have added?